x/pool-incentives/keeper/keeper.go

```diff
@@ -255,6 +256,8 @@ func (k Keeper) GetPoolIdFromGaugeId(ctx sdk.Context, gaugeId uint64, lockableDu
 	bz := store.Get(key)
 
 	if len(bz) == 0 {
+		telemetry.IncrCounter(1, types.NoPoolIdForGaugeTelemetryName)
+		ctx.Logger().Error(types.NoPoolIdForGaugeTelemetryName, "gaugeId", gaugeId, "duration", lockableDuration)
```

Comment:
@alessandrolomanto 

1. Let's make an alert for this counter please `pool_incentives_no_pool_id_for_gauge`. No dashboards needed

Reply:
What values can the counter take: 0 or 1? Should the alert be triggered when it differs from 0?
For this case I prefer to create alert based on logs

Reply:
Based on logs works for me!